waspc/data/Generator/templates/server/src/auth/providers/email/requestPasswordReset.ts

```diff
@@ -24,37 +27,48 @@ export function getRequestPasswordResetRoute({
     return async function requestPasswordReset(
         req: Request<{ email: string; }>,
         res: Response,
-    ): Promise<Response<{ success: true } | { success: false; message: string }>> {
-        const args = req.body || {};
+    ): Promise<Response<{ success: true }> | void> {
+        const args = req.body ?? {};
         ensureValidEmail(args);
 
-        args.email = args.email.toLowerCase();
+        const authIdentity = await findAuthIdentity(
+            createProviderId("email", args.email),
+        );
 
-        const user = await findUserBy({ email: args.email });
-    
         // User not found or not verified - don't leak information
-        if (!user || !user.isEmailVerified) {
+        if (!authIdentity) {
             await doFakeWork();
             return res.json({ success: true });
         }
 
-        if (!isEmailResendAllowed(user, 'passwordResetSentAt')) {
-            return res.status(400).json({ success: false, message: "Please wait a minute before trying again." });
+        const providerData = deserializeAndSanitizeProviderData<'email'>(authIdentity.providerData);
+        if (!providerData.isEmailVerified) {
+            await doFakeWork();
```

Comment:
At some point we decided that, I can't remember 🤷  I guess it made sense if they didn't verify their email, they can't request password resets. 

If they made an account and forgot about it, they can still signup since the email not verified.

Reply:
I've decided to remove this piece of code since it made sense to allow for password reset 👍

Reply:
Sounds good!